lib/webrtc/call.js

```diff
@@ -712,7 +784,7 @@ MatrixCall.prototype._onAddStream = function(event) {
         t.onstarted = hookCallback(self, self._onRemoteStreamTrackStarted);
     });
 
-    event.stream.onended = hookCallback(self, self._onRemoteStreamEnded);
+    event.stream.oninactive = hookCallback(self, self._onRemoteStreamEnded);
```

Comment:
You didn't mention about changing this in the PR description, what's the deal? I'm guessing they changes `onended` to `oninactive`, and have deprecated `onended`? Should we be checking for the presence of `oninactive` to accomodate older browsers?